docs/usage/configuration-options.md

```diff
@@ -2231,8 +2231,8 @@ For example, the following package rule can be used to replace the registry for
   "packageRules": [
     {
       "matchDatasources": ["docker"],
-      "matchPackagePatterns": ["^docker.io/.*)"],
-      "replacementNameTemplate": "{{{replace 'docker.io/' 'ghcr.io/' packageName}}}"
+      "matchPackagePatterns": ["^docker\\.io/.+"],
+      "replacementNameTemplate": "{{{replace 'docker\\.io/' 'ghcr.io/' packageName}}}"
```

Comment:
Ah interesting.... 

would using `matchPackagePrefixes` instead avoid needing to escape the `.`?